templates/base/_package.json

```diff
@@ -2,12 +2,20 @@
   "name": "{{name}}",
   "version": "1.0.0",
   "description": "{{name}} React component library",
-  "main": "dist/cjs/index.js",
-  "module": "dist/es/index.js",
+  "main": "dist/cjs/prod/index.js",
```

Comment:
`main` and `module` are couple of legacy keys. Modern bundlers and NodeJS >= 14 will use `exports`, which then conditionally load bundles. If anything fails, we have `default` and legacy keys as fallback.

[Webpack info](https://webpack.js.org/guides/package-exports/#conditions) | [NodeJS info](https://nodejs.org/api/packages.html#packages_conditional_exports)